pkg/controller/perconaservermongodb/version_test.go

```diff
@@ -307,3 +328,495 @@ func Test_majorUpgradeRequested(t *testing.T) {
 		})
 	}
 }
+
+func TestVersionMeta(t *testing.T) {
+	q, err := resource.ParseQuantity("1Gi")
+	if err != nil {
+		t.Fatal(err)
+	}
+	tests := []struct {
+		name        string
+		cr          api.PerconaServerMongoDB
+		want        VersionMeta
+		clusterWide bool
+		helmDeploy  bool
+	}{
+		{
+			name: "Test 1",
```

Comment:
Pls, give tests a more descriptive name, explaining what is being tested.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1139/commits/1242f8713083265eae5517c5894c8332bd20c0ff